sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionCatalog.scala

```diff
@@ -75,6 +77,33 @@ private[sql] class HiveSessionCatalog(
   // | Methods and fields for interacting with HiveMetastoreCatalog |
   // ----------------------------------------------------------------
 
+  override def validateName(name: String): Boolean = {
+    super.validateName(name)
```

Comment:
Ok, will remove it. My original idea is to enforce the name validation rules of Spark, even if, in the future, the `MetaStoreUtils.validateName` has different rules.